packages/ngx-qgrid-plugins/src/lib/data-provider/data-provider.component.ts

```diff
@@ -0,0 +1,46 @@
+import { HttpClient } from '@angular/common/http';
+import {ChangeDetectionStrategy, Component, EventEmitter, Input, OnChanges, OnInit, Output} from '@angular/core';
+import { GridPlugin, StateAccessor } from '@qgrid/ngx';
+import { PipeUnit } from '@qgrid/core/public-api';
+
+@Component({
+	selector: 'q-data-provider',
+	template: '',
+	providers: [GridPlugin, StateAccessor],
+	changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class DataProviderComponent implements OnInit, OnChanges {
+	nextReference: (args) => void;
```

Comment:
just rename it to `next` and make it private please, also rename `args` to `data` with type `any[]`